data/projects.json

```
@@ -1,4 +1,14 @@
 [
+    {
+        "name": "acetic",
+        "display": "acetic",
+        "description": "online audio distribution platform and music sharing website, that enables its users to upload, promote, and share audio, as well as a digital signal processor enabling listeners to stream audio",
+        "author": "HassanBahati",
+        "category": "web-app, music ",
+        "level": "intermediate",
+        "site": "https://acetic-tunes.netlify.app/",
+        "tags": "tailwindcss, reactjs"
```

To match with the idea please add the tag `music-sharing`
```suggestion
        "tags": "music-sharing, tailwindcss, reactjs"
```